packages/eslint-config/configs/typescript.js

```diff
@@ -43,6 +45,8 @@ module.exports = {
       },
     ],
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/key-spacing': 'off',
+    '@typescript-eslint/lines-around-comment': 'off',
```

Comment:
Formatting rules, which is recommended to disable if we have other formatting rules:

- https://typescript-eslint.io/rules/block-spacing/
- https://typescript-eslint.io/rules/consistent-generic-constructors
- https://typescript-eslint.io/rules/key-spacing
- https://typescript-eslint.io/rules/lines-around-comment

We already have the base-eslint rules (e.g. `block-spacing` et. al.) disabled, so this stays consistent with what we already have in place.

Reply:
aren't these disabled automatically with the prettier plugin? 🤔